src/Presentation/AmountFormatter.php

```diff
@@ -14,8 +14,8 @@ class AmountFormatter {
 	private $locale;
 
 	private $localeData = [
-		'de_DE' => [2, ',', '' ],
-		'en_US' => [2, '.', '' ]
+		'de_DE' => [2, ',', ''],
+		'en_US' => [2, '.', '']
```

Comment:
ah, yeah, right, sorry. I've actually submitted a code with wrong spaced stuff in array this week and wondered why the sniffer let it through. It was some two days ago, apparently far enough for me to forget it completely.